spec/views/collections/_show_descriptions.html.erb_spec.rb

```diff
@@ -13,6 +13,7 @@
     end
     before do
       assign(:presenter, presenter)
+      allow(Sufia::CollectionSizeService).to receive(:run).and_return(123456678)
```

Comment:
Also, don't ever let our users see this number or they'll get funny ideas about how much they can upload. :wink: